crates/turborepo-telemetry/src/events/generic.rs

```diff
@@ -134,8 +134,35 @@ impl GenericEventBuilder {
         self
     }
 
-    // run data
+    // args
+    pub fn track_arg_usage(&self, arg: &str, is_set: bool) -> &Self {
+        self.track(Event {
+            key: format!("arg:{}", arg),
+            value: if is_set { "set" } else { "default" }.to_string(),
+            is_sensitive: EventType::NonSensitive,
+        });
+        self
+    }
 
+    pub fn track_arg_remote_cache_timeout(&self, timeout: u64) -> &Self {
+        self.track(Event {
+            key: "arg:remote-cache-timeout".to_string(),
+            value: timeout.to_string(),
+            is_sensitive: EventType::NonSensitive,
+        });
+        self
+    }
```

Comment:
We could write a generic `track_arg_value` function to avoid some of the code dupe. Possibly also take a `is_sensitive` bool and avoid sending the event if `true`?
```suggestion
    pub fn track_arg_value(&self, arg: &str, value: impl Display) -> &Self {
        self.track(Event {
            key: format!("arg:{arg}"),
            value: value.to_string(),
            is_sensitive: EventType::NonSensitive,
        });
        self
    }
```